packages/epics/src/profile/components/person-head.tsx

```diff
@@ -0,0 +1,189 @@
+import { MemberType } from '@hypha-platform/graphql/rsc';
+import { Text } from '@radix-ui/themes';
+import {
+  Image,
+  Card,
+  Avatar,
+  AvatarImage,
+  Container,
+  Button,
+  Skeleton,
+} from '@hypha-platform/ui';
+import {
+  Share2Icon,
+  CopyIcon,
+  Pencil1Icon,
+  LinkedInLogoIcon,
+  Link2Icon,
+} from '@radix-ui/react-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXTwitter } from '@fortawesome/free-brands-svg-icons';
+
+interface Socials {
+  LinkedIn: string;
+  X: string;
+  Website: string;
+}
+
+interface PersonHeadProps {
+  isLoading: boolean;
+  about: string;
+  background: string;
+  socials: Socials;
+}
+
+export const PersonHead = ({
+  isLoading,
+  avatar = '/placeholder/space-avatar-image.png',
+  name,
+  surname,
+  about,
+  background = '/placeholder/space-lead-image.png',
+  socials,
+}: PersonHeadProps & MemberType) => {
+  const customLogoStyles: React.CSSProperties = {
+    width: '128px',
+    height: '128px',
+    position: 'absolute',
+    bottom: '-35px',
+    left: '15px',
+  };
+
+  return (
+    <div className="flex">
+      <Container>
+        <Card className="relative">
+          <Skeleton height={270} width={768} loading={isLoading}>
+            <Image
+              width={768}
+              height={270}
+              className="rounded-xl max-h-[270px] w-full object-cover"
+              src={background}
+              alt={'Profile Lead Image:' + name + surname}
```

Comment:
Let's use template literals 🙏
```suggestion
              alt={`Profile Lead Image: ${name}  ${surname}`}
```